sb_telemetry_util/telemetryService.js

```diff
@@ -295,4 +295,31 @@ telemetryService.prototype.generateApiCallLogEvent = function (data) {
   })
 }
 
+/**
+ * This function used to generate api_ERROR log event
+ */
+/**
+ * This function used to generate api_ERROR log event
+ */
+telemetryService.prototype.getTelemetryAPIError = function (data, res, context) {
```

Comment:
Follow the funcitonal comments properly for JSdoc 
ex:
/**
 * This function used to get telemetry ERROR event edata
 * @param {Object} data
 */